package.json

```diff
@@ -6,16 +6,21 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
-    "body-parser": "~1.15.1",
-    "cookie-parser": "~1.4.3",
-    "debug": "~2.2.0",
-    "express": "~4.13.4",
-    "googleapis": "^17.1.0",
```

Comment:
Just because you didn't need googleapis doesn't mean that other endpoints didn't need it. (e.g. youtube).